resources/default_pipeline_environment.yml

```diff
@@ -356,6 +356,7 @@ steps:
     install: true
   npmExecuteEndToEndTests:
     runScript: 'ci-e2e'
+    dockerImage: 'ppiper/node-browsers:18-bullseye'
```

Comment:
Ah this is an image specific to us. We don't want it in the piper defaults, right? How can we make it so we can still set the image from _our_ custom defaults?

Reply:
Does it work if we just remove this?

Reply:
That is just a default value. We can override it by defining it in our custom defaults.

Reply:
Ah okay good. I think we should remove it from here so we don't mess with anyone else potentially using this step

Reply:
Yeah, it should AFAIK. I just wanted to set different image (node browser) when `npmExecuteEndtoEndTests` is used, if not set, will take default value from `npmExecuteScripts` i.e node:lts-stretch.

Reply:
It works if this default is gone as well. Do you want to still keep this here? I still think it would be better if it was removed, because we're adding a default for everyone that doesn't need to be there

Reply:
ok, done